src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working.");

```

Comment:
The message 'Truck started working.' should be more informative. For instance, it could include the type of machine that has started working. Consider revising it to something like 'Truck is now working.' to clearly indicate the type of machine.